cmd/controller-manager/app/controller-manager.go

```diff
@@ -98,6 +99,8 @@ member clusters and do the necessary reconciliation`,
 	flags.StringVar(&kubeFedConfig, "kubefed-config", "", "Path to a KubeFedConfig yaml file. Test only.")
 	flags.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 	flags.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+	flags.IntVar(&restConfigQPS, "rest-config-qps", 5, "QPS of rest config.")
```

Comment:
- if QPS is a float, it is better to use `Float32Var` instead.
- I suggest to use more detailed message for the description. E.g.:
  `QPS indicates the maximum QPS to the api-server from this client`
  `Maximum burst for throttle`

Reply:
@xunpan Done. Please review it again. Thanks.